sendsms/backends/ovhsms.py

```diff
@@ -0,0 +1,83 @@
+
+try:
```

Comment:
Don't get this as me reviewing your PR. I don't have the rights to merge 😄, but is there are particular reason as to why you used try function before the imported libs?